modules/hw-installing-amq-interconnect-messaging-bus.adoc

```diff
@@ -34,18 +34,18 @@ amq-interconnect-645db76c76-k8ghs       1/1     Running   0          23h
 interconnect-operator-5cb5fc7cc-4v7qm   1/1     Running   0          23h
 ----
 
-. Verify that the required `baremetal-event-relay` hardware event producer pod is running in the `openshift-baremetal-events` namespace:
+. Verify that the required `bare-metal-event-relay` hardware event producer pod is running in the `openshift-bare-metal-events` namespace:
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-baremetal-events
+$ oc get pods -n openshift-bare-metal-events
 ----
 +
 .Example output
 [source,terminal]
 ----
-NAME                                                         				 READY   STATUS    RESTARTS   AGE
-baremetal-event-relay-controller-manager-b889bb8d5-qwhvs   2/2     Running   0          25s
+NAME                                                        READY   STATUS    RESTARTS   AGE
+bare-metal-event-relay-controller-manager-b889bb8d5-qwhvs   2/2     Running   0          25s
```

Comment:
As far as I know, the pod name would not change. Both versions here were already updated, do we have a record from original doc before naming change? It should be reverted to that.

Reply:
@yliu127 I found a PDF build from the beginning of May. In there, it was `baremetal-hardware-event-proxy-operator-controller-manager-b889bb8d5-qwhvs`

Reply:
`hw-event-proxy-controller-manager-b889bb8d5-qwhvs` from https://github.com/redhat-cne/hw-event-proxy-operator/blob/main/bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml#L240?

Reply:
this would stay as:
hw-event-proxy-controller-manager-b889bb8d5-qwhvs